doc/Building.md

```diff
@@ -65,6 +65,13 @@ sudo apt-get install autoconf automake build-essential ca-certificates cmake \
     lua5.1 luarocks meson nasm ninja-build patch perl pkg-config unzip wget
 ```
 
+**Note:** Debian distributions might need `meson` to be installed from `bookworm-backports`) because the one provided by the default repositories is quite old:
```

Comment:
```suggestion
**Note:** Debian distributions might need `meson` to be installed from `bookworm-backports`) because the version provided by the default repositories is too old:
```